helpers/form/ElementMapFactory.php

```diff
@@ -56,13 +57,79 @@ class ElementMapFactory extends ConfigurableService
     /** @var core_kernel_classes_Resource */
     private $instance;
 
+    /** @var ?bool */
```

Comment:
As I see, you are not adding `null` comment to other class properties, but they are also may be nullable - please, keep the same style.
```suggestion
    /** @var bool */
```